auth/server/src/main/java/com/seagates3/service/RequestorService.java

```diff
@@ -94,6 +113,11 @@ static Requestor getRequestor(ClientRequestToken clientRequestToken)
         }
 
         validateRequestor(requestor, clientRequestToken);
+        GlobalData globalDataObj =
```

Comment:
when cache timeout is 0 can we skip adding them to global data

Reply:
Fixed